### DISCUSSION TITLE
Only support RP2040-LCD-1.28? Could you  support RP2040-Touch-LCD-1.28

### DISCUSSION THREAD

**Discussion by SivaLeeWuHan** [dawigit/picoclock#6] [General]
i have RP2040-Touch-LCD-1.28 and want to try it.

**Comment by dawigit** [comment]
Hi SivaLeeWuHan

I don't have a RP2040-Touch-LCD-1.28 - so sorry, no, not yet.
But I'll support it when I get/buy one.
I've just ordered 2 pcs RP2040-TOUCH-LCD-1.28.

use 'uf2/rp2040-tlcd-128-watch.uf2'
